west.yml

```diff
@@ -65,7 +65,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 60cd202d25ddbf615f9e542e91eb2b4effb06bcf
```

Comment:
This will require sdk-zephyr to be merged first, then this sdk-nrf PR will need to be updated with the new sha and then merged once that is green. In this time window you can't bump sdk-zephyr reference through any other PR as it will lack the nrf-regtool 9.0.0 version in the toolchain, only this one will have it, making it a blocker for all sdk-zephyr manifest changes. Is this is needed because of the major bump of nrf-regtool?

Reply:
Good point, and not really, I can create a separate PR for the requirements bump to circumvent this

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/21152